blazor/datagrid/data-binding.md

```diff
@@ -408,6 +408,8 @@ The sample response object should look like this:
 }
 ```
 
+> The data source is returned in the form of items and count pairs while using the WebAPI Adaptor. But when the [Offline](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DataManager.html#Syncfusion_Blazor_DataManager_Offline) property of [SfDataManager](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.SfDataManager.html) is enabled, the entire data source is returned from the server as a collection of objects. So, the `$inlinecount` will not be present when the `Offline` property is enabled. 
```

Comment:
insert at the last. and also only one request will be made to fetch entire details from the server and no further request will be sent to server.